lib/package-1-12-themes.js

```diff
@@ -21,7 +21,20 @@ function Package( files, runtime ) {
 	Package1_12.apply( this, arguments );
 }
 
-extend( Package.prototype, Package1_12.prototype );
+// Copy some properties from the Package1_12's prototype
+[
+	"AUTHORS.txt",
+	"LICENSE.txt",
+	"images"
+].concat(
+	Object.keys(Package1_12.prototype)
+		.filter( function( property ) {
+			return /\.css$/.test( property );
+		} )
+).forEach( function( prop ) {
+	Package.prototype[ prop ] = Package1_12.prototype[ prop ];
+});
```

Comment:
How big would it be to explicitly list all the properties? Explicit code is easier to understand and maintain... Something like:

```js
extend( Package.prototype, {
  	"AUTHORS.txt": Package1_12[ "AUTHORS.txt" ],
  	...
  	"themes": function( callback ) {
```